api/server/router/container/container_routes.go

```diff
@@ -446,10 +428,6 @@ func (s *containerRouter) wsContainersAttach(ctx context.Context, w http.Respons
 	}
 	containerName := vars["name"]
 
-	if !s.backend.Exists(containerName) {
-		return derr.ErrorCodeNoSuchContainer.WithArgs(containerName)
```

Comment:
Ok, I've got something -- and gets rid of the http stuff from the daemon side, and makes the websocket attach use the same function as the http one.